corehq/motech/fhir/models.py

```diff
@@ -298,3 +303,29 @@ def get_resource_type_or_none(case, fhir_version) -> Optional[FHIRResourceType]:
         )
     except FHIRResourceType.DoesNotExist:
         return None
+
+
+class FHIRImporter(models.Model):
+    domain = models.CharField(max_length=127, db_index=True)
+    connection_settings = models.ForeignKey(
+        ConnectionSettings,
+        on_delete=models.PROTECT,
+    )
+    fhir_version = models.CharField(
+        max_length=12,
+        choices=FHIR_VERSIONS,
+        default=FHIR_VERSION_4_0_1,
+    )
+    frequency = models.CharField(
+        max_length=12,
+        choices=IMPORT_FREQUENCY_CHOICES,
+        default=IMPORT_FREQUENCY_DAILY,
+    )
```

Comment:
Could call this a YAGNI until there are more options.

Reply:
Yeah ... maaaybe. I'm kinda tempted to add "weekly" and "monthly" so we have parity with the OpenMRS version.